### PR TITLE
Improve wording of callbacks and promises exercises

### PR DESCRIPTION
**Pull request by 40thieves**
Does what it says on the 🛢 In response to students' feedback

cc @djgrant

### CODE REVIEW

File: week-5/B-callbacks/exercise.js [hunk 1]
```diff
@@ -21,11 +24,14 @@ function exercise1() {
   EXERCISE 2
   =======
 
-  The function provided for you (functionCallsBack) will call the callback
-  with a result
+  A function is provided for you (functionCallsBack) that will call the first
```
Comment (djgrant) [inline week-5/B-callbacks/exercise.js]: Actual function is called `functionThatCallsBack` not functionCallsBack

File: week-5/B-callbacks/exercise.js [hunk 2]
```diff
@@ -37,15 +43,22 @@ function exercise2(result) {
   EXERCISE 3
   =======
 
-  Write a function that calls the callback with the parameter "Hello from the
+  The exercise3 function will be called for you automatically. It is called
+  with a callback function as the first parameter. Call this callback function
+  (hint: use () parentheses) with the parameter "Hello from the callback" and
```
Comment (djgrant) [inline week-5/B-callbacks/exercise.js]: I like to put code syntax inside back ticks to attempt to distinguish from English syntax.

File: week-5/B-callbacks/exercise.js [hunk 3]
```diff
@@ -37,15 +43,22 @@ function exercise2(result) {
...
+  it will insert it into the #exercise3 element.
+
+  EXPECTED RESULT: The #exercise3 element has textContent = "Hello from the
   callback"
 
-  What happens to the #exercise3 element? Why does this happen? (Hint: look at
-  line 63 below)
+  Can you explain why we get this result? Write down your explanation in a
+  comment below (Hint: look at line 63 below)
```
Comment (djgrant) [inline week-5/B-callbacks/exercise.js]: I think line 63 is already out of date! Maybe something like "Look at the comment inside the function"

File: week-5/B-callbacks/exercise.js (hunk 1 above)
Comment (40thieves) [reply week-5/B-callbacks/exercise.js]: Good catch. Thanks

File: week-5/B-callbacks/exercise.js (hunk 3 above)
Comment (40thieves) [reply week-5/B-callbacks/exercise.js]: You're right that it was out of date, but actually I was referencing the `exercise3` callsite (now on line 77). Updated to not use a line number.

**Comment by 40thieves** [comment]
Thanks @djgrant! Pushed a commit to address the feedback

**Comment by 40thieves** [comment]
Hopefully you're happy with those fixes @djgrant. Merging

**Pull request merged**